lib/smart_answer_flows/marriage-abroad/outcomes/outcome_ceremonies_in_dominican_republic.govspeak.erb

```diff
@@ -57,7 +57,7 @@
                locals: { calculator: calculator } %>
     <%= render partial: 'link_to_consular_fees.govspeak.erb',
                locals: { calculator: calculator } %>
-
-    <%= render partial: 'pay_by_cash_or_credit_card_no_cheque.govspeak.erb' %>
+    <%= render partial: 'payment_for_services.govspeak.erb',
+               locals: { calculator: calculator } %>
   <% end %>
 <% end %>
```

Comment:
@floehopper:

The `marriage-abroad-countries-leading-to-outcome.rb` script allows me to see a list of countries that lead a specific outcome. That hasn't (or at least shouldn't have) changed in any of these commits.

The `add-payment-partials-to-marriage-abroad-services-data.rb` script was changed in a number of these commits. I used this temporary script to update marriage_abroad_services.yml data in the commits that dealt with outcomes that could be reached by more than country. In the reworked version of this branch (PR #2507), I've consistently updated and used the `add-payment-partials-to-marriage-abroad-services-data.rb` script in each commit that amends the marriage_abroad_services.yml data. Hopefully that makes its purpose and use a bit clearer!